tfx/examples/cifar10/distributed/README.md

```diff
@@ -0,0 +1,86 @@
+# CIFAR-10 Transfer Learning and MLKit integration Example with Distributed Training
+
+This example illustrates how to modify the base example for distributed training with a distributed node pool on Google Kubernetes Engine(GKE).
+
+## Instructions
+
+### Set Up a Node Pool
+The guide assumes that you have command line access to `kubectl`.
+If you do not have access to a GKE cluster, follow the quickstart guide [here](https://cloud.google.com/kubernetes-engine/docs/quickstart) to start one.
+
+Then, follow the instructions below for how to set up a node pool on GKE:
+https://cloud.google.com/kubernetes-engine/docs/how-to/node-pools
+
+If you wish to use GPUs in your training, you may follow this guide instead:
+https://cloud.google.com/kubernetes-engine/docs/how-to/gpus
+
+### Set Up a Custom Image for Training (Optional)
+The base cifar10 example relies on dependencies including TensorFlowJS that do not come with the default TFX installation. 
+If you do not wish to use a custom docker image, you can remove the TFLite model rewriting portion in `cifar10_utils_native_keras`.
+
+Start from a base tensorflow gpu image in Docker:
+
+```
+docker run -it tensorflow/tensorflow:latest-gpu
+```
+
+Then, follow the instructions in the base example to configure your training environment. Once you are done, commit the image with:
+
+```
+docker commit [CONTAINER_ID] [new_image_name]
```

Comment:
This is a discouraged way to build containers.  Could you instead provide the contents of a Dockerfile inline?

Reply:
Yes, I changed the instructions to build from a Dockerfile.